.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,31 +9,18 @@ labels: 'type:bug, status:requirements, priority-5-triage'
       Email renovate-disclosure@whitesourcesoftware.com instead.
 -->
 
-**What Renovate type, platform and version are you using?**
-
-<!-- Tell us if you're using the hosted App, or if you are self-hosting Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
-
 **Renovate type**
 
 - [ ] Hosted app
-- [ ] Self hosted
-
-**Renovate platform**
-
-- [ ] Azure DevOps (dev.azure.com)
-- [ ] Azure DevOps Server (self-hosted)
-- [ ] Bitbucket Cloud (bitbucket.org)
-- [ ] Bitbucket Server (self-hosted)
-- [ ] Gitea (gitea.com, self-hosted)
-- [ ] GitHub (github.com, GitHub Enterprise)
-- [ ] GitLab (gitlab.com, self-hosted)
```

Comment:
I see you've removed the platform checkboxes (Azure, Bitbucket, etc.), what is the reason that you're removing those?

My reasoning to have the checkboxes is to make it easy to find the supported platforms, and the ways in which that platform can be used.

Reply:
Added back